pkg/expression/expression.go

```diff
@@ -0,0 +1,41 @@
+package expression
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/antonmedv/expr"
+)
+
+type Expression struct {
+	Comparison Comparison `yaml:"comparison" mapstructure:"comparison"`
+}
+
+type Comparison struct {
+	Key      any    `yaml:"key" mapstructure:"key"`
+	Operator string `yaml:"operator" mapstructure:"operator"`
+	Value    any    `yaml:"value" mapstructure:"value"`
+}
+
+func (e Expression) Evaluate() (any, error) {
```

Comment:
why not just returning `bool` on this instead of `any`?
we always do boolean comparison right?

Reply:
right now we just have comparison operator implemented. but later, when we would incorporate other operators then output can be any. this implemenation is to keep this compatile.
 
here is the list of operators https://expr-lang.org/docs/language-definition#operators

Reply:
Even so, I don't think we need this a single struct under Expression.
We could just pass the expression as string and the expression package could just decide the logic later.

Reply:
But for shield case, do we need other output rather than boolean? if it is in permission, we only have results, allowed or not allowed right?

Reply:
Regardless of that, since this is not part of the schema itself, I guess it is fine to return boolean here. 
We could change later the return value if needed.

Reply:
yes I agree mas, chaning this to boolean